spec/security.md

```diff
@@ -1,4 +1,64 @@
 Security Considerations {#sec-security}
 =====================
 
-**TODO**(shaseley): Fill this in.
+<div class="non-normative">
+*This section is non-normative.*
+
+The main security consideration for the APIs defined in this specification is
+whether or not any information is potentially leaked between origins by
+timing-based side-channel attacks.
+</div>
+
+`postTask` as a High-Resolution Timing Source {#sec-security-high-res-timer}
+---------------------
+
+<div class="non-normative">
+The first consideration is whether the {{Scheduler/postTask()}} API can be used
+as a high-resolution timing source. {{Scheduler/postTask()}}'s
+{{SchedulerPostTaskOptions/delay}}, like
+{{WindowOrWorkerGlobalScope/setTimeout()}}'s timeout value, is expressed in
+whole milliseconds (the minimum non-zero delay being 1 ms), and there is no
+guarantee that tasks will run exactly when the delay expires since tasks are
+queued when the delay expires. Given this, we do not believe the API can be
+used as a high-resolution timing source, but we mention it here because of the
```

Comment:
Try to lead with the lede. So something like,

> This API cannot be used as a high-resolution timing source. Like setTimeout's timeout value, postTask()'s delay is expressed in whole milliseconds (the minimum non-zero delay is 1 ms), so callers can't express any timing more precise than 1ms. Further, the fact that tasks are only queued when their delay expires, and not run instantly, further reduces the precision available to callers.

That also gives future API tweakers a clue to look closely at attempts to change those two qualities.